skbio/io/format/tests/test_embed.py

```diff
@@ -25,6 +29,28 @@
     _protein_to_vector, _vector_to_protein
 )
 
+class TestWriteError(TestCase):
+    def test_write_function(self):
+        embed_file = os.path.join('data', 'tiny_embedding_file.npz')
+        embeddings = np.load(embed_file, allow_pickle=False)
```

Comment:
Same here. Also make sure to add comments here describing the file
```suggestion
        embed_file = get_data_path('tiny_embedding_file.npz')
        embeddings = np.load(embed_file, allow_pickle=False)
```